pom.xml

```diff
@@ -1060,12 +1060,12 @@ Applications based on MyCoRe use a common core, which provides the functionality
       <dependency>
         <groupId>org.apache.pdfbox</groupId>
         <artifactId>pdfbox</artifactId>
-        <version>2.0.17</version>
+        <version>2.0.20</version>
       </dependency>
       <dependency>
         <groupId>org.apache.solr</groupId>
         <artifactId>solr-solrj</artifactId>
-        <version>8.3.1</version>
+        <version>8.5.1</version>
```

Comment:
why not 8.5.2 like in solr runner configuration?

Reply:
it was not suggested by versions plugin. wait a moment ;-)